extensions/agent/src/dialogs/jobDialog.ts

```diff
@@ -542,8 +542,7 @@ export class JobDialog extends AgentDialog<JobData>  {
 	private populateScheduleTable() {
 		let data = this.convertSchedulesToData(this.schedules);
 		this.schedulesTable.data = data;
-		this.schedulesTable.height = 750;
-
+		this.schedulesTable.height = 500;
```

Comment:
And what if the user has the ADS window height less than 500? I'd expect the same issue to occur. Ideally this should be utilizing a flex container where the actual schedules table takes up as much space as it can between the header and the buttons (with a reasonable minimum size of course)

Reply:
Well the way our dialogs work, they don't resize (for example when you resize the window) until the tab is changed. So, even the flex container will face this issue. I took the steps table size since no one has raised an issue for that.

Reply:
You should be able to do something with CSS to make the header and footer take up a specific amount of space and then the table take up the rest. Something like this perhaps https://stackoverflow.com/questions/17141961/css-layout-with-fixed-top-and-bottom-variable-height-middle - although there might be a better way with flex containers.

Potentially could also put the buttons above the grid which would also solve the issue.